webextension/_locales/en_US/messages.json

```diff
@@ -71,6 +71,12 @@
     "message": "Description"
   },
 
+  "titleInitialPromptNoContext": {
```

Comment:
Let's rename this to `titleInitialPromptBasic`, and below, `titleInitialPromptFriendly`, to follow the naming convention used in testing (which I guess you had no way to know about...)

Reply:
Will do. Note that I have added a block of CSS with `display:none` to hide the details that relevant variants should not show, and this was the only bit of actual text left which had to change (which is why there aren't more of these string-changes in the PR).